client/functions.lua

```diff
@@ -71,10 +71,7 @@ end
 
 function QBCore.Functions.RequestAnimDict(animDict)
 	if HasAnimDictLoaded(animDict) then return end
```

Comment:
Might aswell remove this check as ox_lib function has the same check and its just duplicated code at this point.

We only convert these to OX for compatibility and people should just use ox functions natively anyway